### ISSUE TITLE
[Item/Loot] DK tier 8 tanking leggings and set bonus

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#3562]
Fine, here's the new issue.

So the DK tank t8 legs (current ID in game 46121) are now missing its stats and 3 of the DK Tank t8 set have wrong set bonus.
Wrong set bonus IDs in game are 46119, 46122, 46121.

![WoWScrnShot_090319_184109](https://user-images.githubusercontent.com/31132743/64229169-53320680-cee9-11e9-8956-790414ff8d68.jpg)
no stats, wrong set bonus

![tankbonus](https://user-images.githubusercontent.com/31132743/64229185-66dd6d00-cee9-11e9-9c4f-275115ed919d.png)
and this is the intended tanking set bonus for these 3 tank set pieces

**Comment by wow-mania** [comment]
![image](https://user-images.githubusercontent.com/12248683/64244949-e2e3af00-cf01-11e9-80aa-42da67eaeeea.png)
Works perfectly on dev and on red.

**Comment by Dranah** [comment]
ffs jani you are looking at the wrong item. You are checking ID 46116 and I reported ID 46121.

**Comment by Dranah** [comment]
Sigh. Once more, this is meant to be THE TANKING SET.
You have now added DPS set stats to it, and it still has the DPS set bonus, but it should have TANK stats and TANK set bonus.
Now there are two DPS legs in the game, but no tanking t8 for DK.

**How it should work:** This piece of item should be a TANKING item.
It should have THESE stats and set bonus:
![WoWScrnShot_090519_202225](https://user-images.githubusercontent.com/31132743/64368619-ffbfd580-d01a-11e9-9d96-ce48fa20ef24.jpg)

**How it works atm:** This item now has DPS stats and still has the DPS set bonus.

Note: the set bonus needs to be fixed for all of the tanking pieces.

**Comment by wow-mania** [comment]
@kitzunu where is the file?

**Comment by Kitzunu** [comment]
Not fixed, forgot to remove the lable...

**Comment by wow-mania** [comment]
Link me the item from wowhead with the tank stats then

**Comment by wow-mania** [comment]
As- is 
Tank legs is : 46238
DPS legs is: 46121
Do not re-open.

**Comment by Dranah** [comment]
That is fine by me, the only thing that doesn't match is the skin color of the set. However, the set bonus on 2 items of the set is still fucking wrong and you refuse to fix it since 2 reports.

Must I make a 3rd? Specifically for the bonus? Because the first time I reported both item IDs being wrong and you refused to change them. Now you changed the item ID on the leggings at last. But the other 2 items are still wrong ID, wrong bonus, you just fixed the stats.

TL;DR Right now you cannot bloody have 4set t8 tank bonus on a god damn deathknight because the bonus of 2 pieces is wrong. You can only get 3/5 tanking bonus.

**Comment by Dranah** [comment]
And please remove "fixed color of DK t8 set" from the changelog on discord because clearly, the fix didn't work, you just swapped item IDs for a misc item with the correct color but it fucked up the entire set in every other way.
